README.md

```diff
@@ -204,6 +235,36 @@ To run all examples and play around with the code you have to run two separate c
 - `yarn dev` - runs webpack and node dev server
 - `yarn build:dev` - runs rollup in watch mode - livereload if you do changes to the insights library
 
+## Node.js module
```

Comment:
I'm not sure I would have put this in an entirely new section.
The main difference is in the installation process right? 
- in the browser, we use the snippet
- in node, we use whatever dependency management we have

So maybe this should just be a section under `Loading and initializing the library` ?

Reply:
Right. I will move this part under `Loading and ...` and also add how to `npm install` or `yarn install` this.